### PR TITLE
Update Bayesian Neural Network example to v4

### PR DESCRIPTION
**Pull request by fonnesbeck**
Probably needs a few style tweaks still, but otherwise ready for reveiw.

### CODE REVIEW

**Comment by fonnesbeck** [comment]
Pre-commit checks passed locally

File: myst_nbs/variational_inference/bayesian_neural_network_advi.myst.md [hunk 1]
```diff
@@ -378,35 +321,14 @@ For fun, we can also look at the trace. The point is that we also get uncertaint
...
-<iframe width="560" height="315" src="https://www.youtube.com/embed/LlzVlqVzeD8" frameborder="0" allowfullscreen></iframe>
-
-Finally, you can download this NB [here](https://github.com/twiecki/WhileMyMCMCGentlySamples/blob/master/content/downloads/notebooks/bayesian_neural_network.ipynb). Leave a comment below, and [follow me on twitter](https://twitter.com/twiecki).
+You might argue that the above network isn't really deep, but note that we could easily extend it to have more layers, including convolutional ones to train on more challenging data sets.
 
 ## Acknowledgements
 
 [Taku Yoshioka](https://github.com/taku-y) did a lot of work on ADVI in PyMC3, including the mini-batch implementation as well as the sampling from the variational posterior. I'd also like to the thank the Stan guys (specifically Alp Kucukelbir and Daniel Lee) for deriving ADVI and teaching us about it. Thanks also to Chris Fonnesbeck, Andrew Campbell, Taku Yoshioka, and Peadar Coyle for useful comments on an earlier draft.
 
 +++
```
Comment (OriolAbril) [inline myst_nbs/variational_inference/bayesian_neural_network_advi.myst.md:331]: add authors section. This allows to keep the link to the original blog post (even if updated multiple times and it becomes harder to recognize the origin) as well as listing authors with their names as clickable links to their github profile, twitter...

File: myst_nbs/variational_inference/bayesian_neural_network_advi.myst.md [hunk 2]
```diff
@@ -378,35 +321,14 @@ For fun, we can also look at the trace. The point is that we also get uncertaint
...
 
-(c) 2017 by Thomas Wiecki, updated by Maxim Kochurov
-
-Original [blog post](https://twiecki.github.io/blog/2016/06/01/bayesian-deep-learning/).
-
-+++
-
 ## References
 
 :::{bibliography}
```
Comment (OriolAbril) [inline myst_nbs/variational_inference/bayesian_neural_network_advi.myst.md:334]: can't comment any lower, add the page footer: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#epilogue

**Comment by twiecki** [comment]
@fonnesbeck Any progress?

**Comment by twiecki** [comment]
@fonnesbeck Not sure how but there are conflicts.

**Comment by fonnesbeck** [comment]
Yeah, no idea. Might be easiest to close and create a fresh PR, unless @OriolAbril has any ideas.

**Comment by twiecki** [comment]
Probably easier to reopen.

**Pull request closed without merging**